frontend/src/App.js

```diff
@@ -13,14 +15,14 @@ function App() {
       <main>
         <Container>
           <Routes>
-          < Route path="/news" element = {<NewsScreen />}/>
-          < Route path="/settings" element = {<SearchSettings />}/>
-
+            <Route path="/news" element={<NewsScreen />} />
+            <Route path="/settings" element={<SearchSettings />} />
+            <Route path="/signin" element={<SignIn />} />
```

Comment:
I think that we shouldn't include Header component in login page (redirections to user's Search Settings and News)